mftf/2.3/getting-started.md

```diff
@@ -204,35 +198,110 @@ vendor/bin/mftf run:test AdminLoginTest --remove
 
 See more commands in [`mftf`][].
 
-### Step 9. Generate reports    {#reports}
+### Step 7. Generate reports    {#reports}
 
 During testing, the MFTF generates test reports in CLI.
 You can generate visual representations of the report data using [Allure Framework][].
 To view the reports in GUI:
 
--  [install Allure][]
--  run the tool to serve the artifacts in `dev/tests/acceptance/tests/_output/allure-results/`:
+- [install Allure][]
+- run the tool to serve the artifacts in `dev/tests/acceptance/tests/_output/allure-results/`:
 
 ```bash
 allure serve dev/tests/acceptance/tests/_output/allure-results/
 ```
 
 Learn more about Allure in the [official documentation][allure docs].
 
+## Set up a standalone MFTF
+
+The MFTF is a root level Magento dependency, but it is also available for use as a standalone application.
+The potential use case is when you develop the MFTF or contribute to it, as it facilitates debugging and tracking changes.
```

Comment:
Suggested edit: You may want to use a standalone application when you develop for or contribute to MFTF, which facilitates debugging and tracking changes.

Reply:
:ok: